src/sections/chat/utils/sortAgents.ts

```diff
@@ -1,31 +1,31 @@
-import orderBy from 'lodash/orderBy';
-import { UserChatResponse } from 'types/user-backend';
-
-const sortAgents = ({ agents, sortType }: { agents?: UserChatResponse[]; sortType: string }) => {
-  switch (sortType) {
-    case 'nameAsc':
-      return orderBy(
-        agents,
-        [(agent) => `${agent.firstName} ${agent.lastName}`.toLowerCase()],
-        ['asc']
-      );
-
-    case 'nameDesc':
-      return orderBy(
-        agents,
-        [(agent) => `${agent.firstName} ${agent.lastName}`.toLowerCase()],
-        ['desc']
-      );
-
-    case 'oldest':
-      return orderBy(agents, ['createdAt'], ['asc']);
-
-    case 'newest':
-      return orderBy(agents, ['createdAt'], ['desc']);
-
-    default:
-      return agents;
-  }
-};
-
-export default sortAgents;
+import orderBy from 'lodash/orderBy';
+import { UserChatResponse } from 'types/user-backend';
+
+const sortAgents = ({ agents, sortType }: { agents?: UserChatResponse[]; sortType: string }) => {
```

Comment:
I have already seen this piece of code src/sections/chat/utils/sortAgents.ts I assume you didn't make branch from develop

Reply:
It's just a linter fixed some

Reply:
if you didnt add this code why it here. its mean that someone done something wrong, I have seen this code 3 time yesterday in different prs, so you need with them figure out what is the issue, send me fix separately and task separatelly